GitTank/GitProcessor.cs

```diff
@@ -156,5 +157,28 @@ public async Task Push()
                 await _processHelper.Execute();
             }
         }
+
+        public async Task OpenTerminal(string selectedRepository)
```

Comment:
it may be better to have a separate class that manages terminal opening

Reply:
I thought about that. Plan to do this soon.